public/app/plugins/panel/gauge/GaugePanel.tsx

```diff
@@ -61,6 +61,7 @@ export class GaugePanel extends PureComponent<PanelProps<GaugeOptions>> {
         source={data}
         renderCounter={renderCounter}
         orientation={VizOrientation.Auto}
+        itemAsSquares={true}
```

Comment:
rather than `itemAsSquares` could VizOrientation have a "square" or "grid" mode?

Reply:
Yes, I was thinking about that one as well. But wasn't sure if we wanted to enable this for some of the visualizations. So basically this is the "auto" mode for the visualizations where it makes sense to display them as squares.

But love the feedback and will change it when I continue on this one. Thanks! 🙌